### PR TITLE
Fix py26 py33 deps

### LINKED ISSUE #4577: awscli-bundle 1.6.256 does not install with Python 2.6 (colorama dependency not met)
On-going issue with colorama dependency.

previous closed issue: https://github.com/aws/aws-cli/issues/4563

```
sudo awscli-bundle/install -i /usr/local/aws -b /usr/local/bin/aws
...
Running cmd: /usr/bin/python virtualenv.py --no-download --python /usr/bin/python /usr/local/aws
Running cmd: /usr/local/aws/bin/pip install --no-cache-dir --no-index --find-links file:///root/aws-install/awscli-bundle/packages/setup setuptools_scm-1.15.7.tar.gz
Running cmd: /usr/local/aws/bin/pip install --no-cache-dir --no-index --find-links file:///root/aws-install/awscli-bundle/packages awscli-1.16.256.tar.gz
Running cmd: /usr/bin/python virtualenv.py --no-download --python /usr/bin/python /usr/local/aws
Running cmd: /usr/local/aws/bin/pip install --no-cache-dir --no-index --find-links file:///root/aws-install/awscli-bundle/packages/setup setuptools_scm-1.15.7.tar.gz
Running cmd: /usr/local/aws/bin/pip install --no-cache-dir --no-index --find-links file:///root/aws-install/awscli-bundle/packages awscli-1.16.256.tar.gz
STDERR: Traceback (most recent call last):
  File "./awscli-bundle/install", line 162, in <module>
    main()
  File "./awscli-bundle/install", line 151, in main
    pip_install_packages(opts.install_dir)
  File "./awscli-bundle/install", line 119, in pip_install_packages
    pip_script, PACKAGES_DIR, cli_tarball))
  File "./awscli-bundle/install", line 49, in run
    p.returncode, cmd, stdout + stderr))
__main__.BadRCError: Bad rc (1) for cmd '/usr/local/aws/bin/pip install --no-cache-dir --no-index --find-links file:///root/aws-install/awscli-bundle/packages awscli-1.16.256.tar.gz': Processing ./awscli-1.16.256.tar.gz
Collecting botocore==1.12.246 (from awscli==1.16.256)
Collecting docutils<0.16,>=0.10 (from awscli==1.16.256)
Collecting rsa<=3.5.0,>=3.1.2 (from awscli==1.16.256)
Collecting s3transfer<0.3.0,>=0.2.0 (from awscli==1.16.256)
Requirement already satisfied: argparse>=1.1 in /usr/local/aws/lib/python2.6/site-packages (from awscli==1.16.256)
Collecting PyYAML<=3.13,>=3.10 (from awscli==1.16.256)
Collecting colorama<0.3.9,>=0.2.5 (from awscli==1.16.256)
DEPRECATION: Python 2.6 is no longer supported by the Python core team, please upgrade your Python. A future version of pip will drop support for Python 2.6
  Could not find a version that satisfies the requirement colorama<0.3.9,>=0.2.5 (from awscli==1.16.256) (from versions: 0.3.9, 0.4.1)
No matching distribution found for colorama<0.3.9,>=0.2.5 (from awscli==1.16.256)
```

### PR DESCRIPTION
**Pull request by jamesls**
* The colorama version range was wrong.  We include 0.3.9 in the bundled installer but had a range of `<0.3.9`.
* The PyYAML range was wrong.  The 3.x versions were the last to support py2.6/py3.3

Fixes #4577.

## Testing

Generated a bundled installer and tested we could install it on all our supported versions.

```
$ ./test-all-versions.sh

Python 2.6.9
Running cmd: /Users/jamessar/.pythonz/pythons/CPython-2.6.9/bin/python virtualenv.py --no-download --python /Users/jamessar/.pythonz/pythons/CPython-2.6.9/bin/python /tmp/verify/here
Running cmd: /tmp/verify/here/bin/pip install --no-cache-dir --no-index --find-links file:///private/tmp/verify/awscli-bundle/packages/setup setuptools_scm-1.15.7.tar.gz
Running cmd: /tmp/verify/here/bin/pip install --no-cache-dir --no-index --find-links file:///private/tmp/verify/awscli-bundle/packages awscli-1.16.257.tar.gz
You can now run: /tmp/verify/here/bin/aws --version
aws-cli/1.16.257 Python/2.6.9 Darwin/18.7.0 botocore/1.12.247


Python 2.7.14
Running cmd: /Users/jamessar/.pythonz/pythons/CPython-2.7.14/bin/python virtualenv.py --no-download --python /Users/jamessar/.pythonz/pythons/CPython-2.7.14/bin/python /tmp/verify/here
Running cmd: /tmp/verify/here/bin/pip install --no-cache-dir --no-index --find-links file:///private/tmp/verify/awscli-bundle/packages/setup setuptools_scm-1.15.7.tar.gz
Running cmd: /tmp/verify/here/bin/pip install --no-cache-dir --no-index --find-links file:///private/tmp/verify/awscli-bundle/packages awscli-1.16.257.tar.gz
You can now run: /tmp/verify/here/bin/aws --version
aws-cli/1.16.257 Python/2.7.14 Darwin/18.7.0 botocore/1.12.247


Python 3.3.6
Running cmd: /Users/jamessar/.pythonz/pythons/CPython-3.3.6/bin/python virtualenv.py --no-download --python /Users/jamessar/.pythonz/pythons/CPython-3.3.6/bin/python /tmp/verify/here
Running cmd: /tmp/verify/here/bin/pip install --no-cache-dir --no-index --find-links file:///private/tmp/verify/awscli-bundle/packages/setup setuptools_scm-1.15.7.tar.gz
Running cmd: /tmp/verify/here/bin/pip install --no-cache-dir --no-index --find-links file:///private/tmp/verify/awscli-bundle/packages awscli-1.16.257.tar.gz
You can now run: /tmp/verify/here/bin/aws --version
aws-cli/1.16.257 Python/3.3.6 Darwin/18.7.0 botocore/1.12.247


Python 3.4.6
Running cmd: /Users/jamessar/.pythonz/pythons/CPython-3.4.6/bin/python virtualenv.py --no-download --python /Users/jamessar/.pythonz/pythons/CPython-3.4.6/bin/python /tmp/verify/here
Running cmd: /tmp/verify/here/bin/pip install --no-cache-dir --no-index --find-links file:///private/tmp/verify/awscli-bundle/packages/setup setuptools_scm-1.15.7.tar.gz
Running cmd: /tmp/verify/here/bin/pip install --no-cache-dir --no-index --find-links file:///private/tmp/verify/awscli-bundle/packages awscli-1.16.257.tar.gz
You can now run: /tmp/verify/here/bin/aws --version
aws-cli/1.16.257 Python/3.4.6 Darwin/18.7.0 botocore/1.12.247


Python 3.5.3
Running cmd: /Users/jamessar/.pythonz/pythons/CPython-3.5.3/bin/python virtualenv.py --no-download --python /Users/jamessar/.pythonz/pythons/CPython-3.5.3/bin/python /tmp/verify/here
Running cmd: /tmp/verify/here/bin/pip install --no-cache-dir --no-index --find-links file:///private/tmp/verify/awscli-bundle/packages/setup setuptools_scm-1.15.7.tar.gz
Running cmd: /tmp/verify/here/bin/pip install --no-cache-dir --no-index --find-links file:///private/tmp/verify/awscli-bundle/packages awscli-1.16.257.tar.gz
You can now run: /tmp/verify/here/bin/aws --version
aws-cli/1.16.257 Python/3.5.3 Darwin/18.7.0 botocore/1.12.247


Python 3.6.8
Running cmd: /Users/jamessar/.pythonz/pythons/CPython-3.6.8/bin/python virtualenv.py --no-download --python /Users/jamessar/.pythonz/pythons/CPython-3.6.8/bin/python /tmp/verify/here
Running cmd: /tmp/verify/here/bin/pip install --no-cache-dir --no-index --find-links file:///private/tmp/verify/awscli-bundle/packages/setup setuptools_scm-1.15.7.tar.gz
Running cmd: /tmp/verify/here/bin/pip install --no-cache-dir --no-index --find-links file:///private/tmp/verify/awscli-bundle/packages awscli-1.16.257.tar.gz
You can now run: /tmp/verify/here/bin/aws --version
aws-cli/1.16.257 Python/3.6.8 Darwin/18.7.0 botocore/1.12.247

Python 3.7.3
Running cmd: /Users/jamessar/.pythonz/pythons/CPython-3.7.3/bin/python virtualenv.py --no-download --python /Users/jamessar/.pythonz/pythons/CPython-3.7.3/bin/python /tmp/verify/here
Running cmd: /tmp/verify/here/bin/pip install --no-cache-dir --no-index --find-links file:///private/tmp/verify/awscli-bundle/packages/setup setuptools_scm-1.15.7.tar.gz
Running cmd: /tmp/verify/here/bin/pip install --no-cache-dir --no-index --find-links file:///private/tmp/verify/awscli-bundle/packages awscli-1.16.257.tar.gz
You can now run: /tmp/verify/here/bin/aws --version
aws-cli/1.16.257 Python/3.7.3 Darwin/18.7.0 botocore/1.12.247

```

### CODE REVIEW

**Comment by lukeseawalker** [comment]
Hi,
any ETA for the fix to be merged?
Thanks

**Comment by jamesls** [comment]
Merged now, this will go out in the next release.

File: setup.py [hunk 1]
```diff
@@ -39,13 +39,13 @@ def find_version(*file_paths):
...
+    install_requires.append('colorama>=0.2.5,<=0.3.9')
 elif sys.version_info[:2] == (3, 3):
-    install_requires.append('PyYAML>=3.10,<=5.2')
+    install_requires.append('PyYAML>=3.10,<=3.13')
     # Colorama removed support for EOL pythons.
-    install_requires.append('colorama>=0.2.5,<0.3.9')
+    install_requires.append('colorama>=0.2.5,<=0.3.9')
 else:
-    install_requires.append('PyYAML>=3.10,<=5.2')
+    install_requires.append('PyYAML>=3.10,<5.2')
```
Comment (tkelman) [inline setup.py]: guess there is no 5.2 yet, but the bound being `<` or `<=` should be consistent between setup.cfg and setup.py shouldn't it?

Comment (jamesls) [reply setup.py]: Yeah good catch, I'll update that

**Pull request merged**